collect/central_collector.go

```diff
@@ -217,15 +227,17 @@ func (c *CentralCollector) Start() error {
 	if maxConcurrency <= 0 {
 		maxConcurrency = 4
 	}
-	c.egAgg = &errgroup.Group{}
-	c.egAgg.SetLimit(maxConcurrency) // we want to limit the number of goroutines that are aggregating trace IDs
+	egKeepAgg := &errgroup.Group{}
+	egDropAgg := &errgroup.Group{}
+	egKeepAgg.SetLimit(maxConcurrency) // we want to limit the number of goroutines that are aggregating trace IDs
```

Comment:
I'm not sure I think that these different concurrency limits should all be the same. I'm especially worried that the keepTraces one might want to be higher, or it's going to block our ability to process traces.

I guess we can mess with it, but it's something to watch out for.